dbms/src/main/java/org/polypheny/db/processing/AbstractQueryProcessor.java

```diff
@@ -1273,99 +1264,6 @@ private Map<Long, List<Long>> extractPartitions( Set<Entity> entities ) {
     }
 
 
-    /**
-     * Traverses all TablesScans used during execution and identifies for the corresponding table all
-     * associated partitions that needs to be accessed, on the basis of the provided partitionValues identified in a LogicalFilter
-     * <p>
-     * It is necessary to associate the partitionIds again with the ScanId and not with the table itself. Because a table could be present
-     * multiple times within one query. The aggregation per table would lead to data loss
-     *
-     * @param alg AlgNode to be processed
-     * @param aggregatedPartitionValues Mapping of Scan Ids to identified partition Values
-     * @return Mapping of Scan Ids to identified partition Ids
-     */
-    private Map<Long, List<Long>> getAccessedPartitionsPerScan( AlgNode alg, Map<Long, Set<String>> aggregatedPartitionValues ) {
```

Comment:
Is this logic now somewhere else?

Reply:
Going through the logs it seems this method was replaced by `extractPartitions` just above the removed method in PR #502.